packages/imagemagick6.rb

```diff
@@ -64,5 +64,12 @@ def self.build
 
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    system "ln -s #{CREW_LIB_PREFIX}/libMagickWand-6.Q16HDRI.so.6 #{CREW_LIB_PREFIX}/libMagickWand-6.Q16.so.6"
+    system "ln -s #{CREW_LIB_PREFIX}/libMagickCore-6.Q16HDRI.so.6 #{CREW_LIB_PREFIX}/libMagickCore-6.Q16.so.6"
+  end
+  
+  def self.remove
```

Comment:
I don't believe we have a self.remove method.  Generally, you won't need to remove files if the files have been tracked.  If you apply the suggestion above, you won't need to do this.

Reply:
Could you briefly explain how files are tracked ? I don't really understand this point and that will help me to be more productive next time.

Reply:
When you add files to the `CREW_BREW_DIR` and subdirectories, these files are tracked for installation and therefore are available for removal as well.  If you get a chance, study the `crew` command to get a better idea of how this works.

Reply:
Thanks !